### ISSUE TITLE
IO verbose level

### ISSUE THREAD

**Issue by mfe** [tuttleofx/TuttleOFX#242]
There's no way right now to just dump Input / Output infos : 
IN : /datas/imageIn.jpg
OUT : /datas/imageOut.png

Debug is too wordy and Warning doesn't dump IO.

**Comment by cchampet** [comment]
Is it fixed ?
Currently we have:
* production / release / debug build
* verbose mode for avreader / avwriter

**Comment by fabiencastan** [comment]
Yes, it's done. There is a verbose option is the API and in sam-do.
